components/Badge/badge.css

```diff
@@ -0,0 +1,101 @@
+/* Badges on Avatar Styles Starts */
+.badges-avatar {
+    display: flex;
+    gap: 1rem;
+}
+
+.avatar-badge {
+    border-radius: 50%;
+    display: flex;
+    gap: 2rem;
+}
+
+.avatar-badge img {
+    border-radius: 50%;
+    height: 5rem;
+    width: 5rem;
+}
+
+.round-badge {
+    height: 0.7rem;
+    width: 0.7rem;
+    border-radius: 50%;
+    bottom: 10%;
+    right: 10%;
+    border: 3px solid white;
+}
+
+/* Styling badge color and position */
+
+.badge-off {
+    position: relative;
+
+}
+
+.badge-off .round-badge {
+    position: absolute;
```

Comment:
specify the position in .round-badge class and specify only background color over here . similarly for badge-on and badge-away too